torch/onnx/_flags.py

```diff
@@ -48,3 +48,8 @@ def _load_boolean_flag(
     this_will="enable torch.export.draft_export as a strategy for capturing models",
     default=False,
 )
+PREFER_DEFERRED_RUNTIME_ASSERTS_OVER_GUARDS: bool = _load_boolean_flag(
+    "TORCH_ONNX_PREFER_DEFERRED_RUNTIME_ASSERTS_OVER_GUARDS",
+    this_will="set prefer_deferred_runtime_asserts_over_guards when calling torch.export",
+    default=True,
```

Comment:
@titaiwangms default is set here

Reply:
True, the default in torch is False. If the arg does what it claims to do though it is just going to not enforce the guards during export but instead make them runtime asserts (which we ignore). I think that’s a reasonable thing to do. I propose that we at least try it and run the benchmarks.